src/ethereum_test_filling_tool/main.py

```diff
@@ -85,6 +87,25 @@ def parse_arguments() -> argparse.Namespace:
             help="removes the folder structure from test fixture output",
         )
 
+        parser.add_argument(
+            "--benchmark",
+            action="store_true",
+            help="logs the timing of the test filler for benchmarking",
+        )
+
+        parser.add_argument(
+            "--max-workers",
+            type=int,
+            help="specifies the max number of workers for the test filler \
+                  set to 1 for serial execution",
+        )
+
+        parser.add_argument(
+            "--no-skip",
+            action="store_true",
+            help="fill all test fillers and don't skip any tests",
+        )
```

Comment:
I think something like `"--force-refill"` would be a bit clearer.

Also adding a explanation of when the tests are skipped would be helpful.

Reply:
Changed to `--force-refill`. Comments below hopefully address the latter.

```python
# Only skip if the fixture file already exists, the module
# has not been modified since the last test filler run, and
# the user does not want to overwrite the fixtures (--overwrite).
if (os.path.exists(path) 
    and not is_module_modified(path, self.options.filler_path, module_path)
    and not self.options.force_refill):
          self.log.debug(f"skipping - {full_name}")
          return
```